devito/stencilkernel.py

```diff
@@ -299,6 +474,32 @@ def argtypes(self):
 }
 
 """
+StencilKernel options
+"""
+options = {
+    'at_squeezer': 3,
+    'at_blocksize': [8, 16, 32]
+}
```

Comment:
nothing particularly solid: they simply looked reasonable sizes given the expected size of the parallel loop and the expected number of threads. And in my initial experiments (in Devito3.0) they resulted in performance improvements, while slightly different block sizes didn't change much the situation. I'll introduce more options in the future.